OpenRA.Test/OpenRA.Test.csproj

```diff
@@ -14,7 +14,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|x86'">
     <DebugSymbols>true</DebugSymbols>
-    <OutputPath>..\</OutputPath>
+    <OutputPath>bin\Debug\</OutputPath>
```

Comment:
The original csproj had output defined in bin/Debug IIRC. I didn't want to heavily change that behaviour.